packages/minter-contracts/ligo/src/english_auction/english_auction_tez.mligo

```diff
@@ -120,10 +126,17 @@ let auction_in_progress (auction : auction) : bool =
 let first_bid (auction : auction) : bool =
   auction.highest_bidder = auction.seller
 
-let valid_bid_amount (auction : auction) : bool =
-  (Tezos.amount >= (auction.current_bid + (percent_of_bid_tez (auction.min_raise_percent, auction.current_bid)))) ||
-  (Tezos.amount >= auction.current_bid + auction.min_raise)                                            ||
-  ((Tezos.amount >= auction.current_bid) && first_bid(auction))
+let dont_return_bid (auction : auction) : bool =
+  first_bid(auction)
+
+#if OFFCHAIN_BID
+  || auction.last_bid_offchain 
+#endif
+
+let valid_bid_amount (auction, bid_amount : auction * tez) : bool =
+  (bid_amount >= (auction.current_bid + (percent_of_bid_tez (auction.min_raise_percent, auction.current_bid)))) ||
```

Comment:
small idea: what if we stored `20%` as `120%` and calculated the minimum increase as `auction.current_bid * auction.min_raise_percent / 100` instead of `auction.current_bid + (auction.current_bid * auction.min_raise_percent) / 100?

Reply:
This would be a nice optimization but I worry it will be confusing for people who have already used the contract, and depend on the storage, and I don't think it would offer a significant enough advantage to warrant a change.